ripple/src/main/webapp/businessdetails.js

```diff
@@ -196,6 +197,17 @@ function bdOnload() {
         }
       }
   getDocByDocId("businessDetails", businessId, lambda);
+
+  var lambda2 = (doc) => {
+        if (doc.exists) {
+          var j;
+          for (j = 0; j < 4; j++) {
+            console.log("tags: " + Object.keys(doc.data().tags).toString());
+            //createCarouselElement(doc.data().galleryBlobKeys[i], "imageGallery");
+          }
+        }
+  }
+  getDocByDocId("businesses", businessId, lambda2);
```

Comment:
nit: you don't really need to assign the anonymous function to a variable. You could pass it inline to getDocByDocId() as the last parameter directly. 
There is extra indentation in the body of the anonymous function. 
Same comments apply to "var lambda"

Reply:
We tried passing it in directly but it looked really messy because the function is really long. Would it be possible for me to rename the variable to something more descriptive instead?